website/src/contribute/committer-guide.md

```diff
@@ -77,15 +95,25 @@ At some point in the review process, the change to the codebase will be
 complete. However, the pull request may have a collection of review-related
 commits that are not meaningful to preserve in the history. The reviewer should
 give the LGTM and then request that the author of the pull request rebase,
-squash, split, etc, the commits, so that the history is most useful. Favor
-commits that do just one thing. The commit is the smallest unit of easy
+squash, split, etc, the commits, so that the history is most useful:
+* Favor commits that do just one thing. The commit is the smallest unit of easy
 rollback; it is easy to roll back many commits, or a whole pull request, but
 harder to roll back part of a commit.
+* Commit messages should tag JIRAs and be otherwise descriptive.
+It should later not be necessary to find a merge or first PR commit to find out what caused a change.
+* Squash the "Fixup!", "Address comments" type of commits that resulted from review iterations.
+
+While it is preferred that authors squash commits after review is complete,
```

Comment:
Should this be in "Merging it!" section below?

Reply:
Done!